### PR TITLE
CLIEN-2906 bug fix for service name change

### PR DESCRIPTION
**Pull request by vmsachin**
1. Changed service name from asld to aerospike-connection-manager.
2. Cleaned make file for php installation

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,7 +12,7 @@ ifeq ($(UNAME_S),Darwin)
 else ifeq ($(UNAME_S),Linux)
     EXTENSION := .so
 	PHP_VERSION := $(shell php -i | grep -Po '(?<=PHP Version => ).*' | uniq)
-    RESTART_COMMAND := sudo systemctl restart php$(PHP_VERSION)-fpm && sudo systemctl restart apache2
+    RESTART_COMMAND := systemctl restart php$(PHP_VERSION)-fpm && systemctl restart apache2
```
Comment (khaf) [inline Makefile:15]: I don't understand why `sudo` has been removed. These commands require escalated privilege.

File: aerospike-connection-manager/Makefile [hunk 2]
```diff
@@ -24,7 +24,7 @@ build:
 	$(GOBUILD) .
 
 run: clean proto
-	$(GOBUILD) -o $(BINARY_NAME) -v .
+	$(GOBUILD) -v .
 	./$(BINARY_NAME) -config-file asld.toml
```
Comment (khaf) [inline aerospike-connection-manager/Makefile:28]: Is the name of the default config file correct? Did we not change all the instances of `asld`?

**Review by khaf — APPROVED** [review]
I added a few comments, but the change seems ok to merge.

File: Makefile (hunk 1 above)
Comment (vmsachin) [reply Makefile:15]: I was trying to build the php client on a bare ubuntu docker (as root), and this caused a problem as it does not understand sudo. In cases where there is permission required we can just run the make file as sudo -> `sudo make [target]`

File: aerospike-connection-manager/Makefile (hunk 2 above)
Comment (vmsachin) [reply aerospike-connection-manager/Makefile:28]: no, the config file and binary name remains asld.toml and asld 
maybe we can change the config file to acm-config.toml

**Pull request merged**